core/templates/dev/head/services/RteHelperServiceSpec.js

```diff
@@ -0,0 +1,58 @@
+// Copyright 2018 The Oppia Authors. All Rights Reserved.
```

Comment:
Hm. Did this get removed in develop? Surprised to see it here -- might want to check. /cc @bansalnitish

Reply:
Yup, it's removed.

Reply:
Whoops! Removed this. Thanks!